Clients/GUI/Scripts/SignUpper.cs

```diff
@@ -0,0 +1,18 @@
+using GUI.GUIModels;
+using System.Net;
+using DTO;
+using System.IO;
+using System.Text.Json;
+using HttpWebRequestWrapperLib;
+
+namespace GUI.Scripts
+{
+    public class SignUpper
```

Comment:
Why the class isn't Static while its only method is?

Reply:
Its only method is static nevertheless. So it _is_ in memory all the time, and static method still can have simultaneous instances of itself

Reply:
Fixed